Configuration/AlCa/python/autoCond.py

```diff
@@ -25,21 +25,19 @@
     'run2_mc_pa'                   :    '131X_mcRun2_pA_v3',
     # GlobalTag for Run2 data reprocessing
     'run2_data'                    :    '141X_dataRun2_v2',
-    # GlobalTag for Run2 data 2018B relvals only: HEM-15-16 fail
-    'run2_data_HEfail'             :    '140X_dataRun2_HEfail_v1',
```

Comment:
this broke wf 136.8642: https://cmssdt.cern.ch/SDT/cgi-bin/logreader/el8_amd64_gcc12/CMSSW_15_1_X_2025-02-19-2300/pyRelValMatrixLogs/run/136.8642_RunJetHT2018BHEfail/step3_RunJetHT2018BHEfail.log#/

Reply:
Thank you @mmusich 
Take home lesson: in the future I should trust more [lxr](https://cmssdt.cern.ch/lxr/search?%21v=CMSSW_15_1_X_2025-02-19-2300&_filestring=&_string=run2_data_HEfail) rather than the [search feature in github](https://github.com/search?q=repo%3Acms-sw%2Fcmssw+run2_data_HEfail&type=code)
I will put it back with a fix PR

Reply:
Removal of 'run2_data_HEfail' reverted with the bugfix PR #47409
Also updated accordingly #47386, the backport of this PR in CMSSW_15_0_X (not merged yet)